CHANGELOG.md

```diff
@@ -54,11 +54,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ##### Hydro Station
 
 - Added: In Inner Alpha Nest South: IBJ method to reach item
```

Comment:
not strictly related to this but can you also add a missing dot at the end of the sentence?

Reply:
Shouldn't bullets not have periods? The whole changelog is inconsistent with this.

Reply:
it is inconcistent, but from what i see, consensus seems to be to end with a period.